libsbom/spdx.py

```diff
@@ -0,0 +1,211 @@
+import logging
+import datetime
+import uuid
+
+from spdx_tools.spdx.writer.json     import json_writer
+from spdx_tools.spdx.writer.tagvalue import tagvalue_writer
+from spdx_tools.spdx.writer.xml      import xml_writer
+from spdx_tools.spdx.writer.yaml     import yaml_writer
+from spdx_tools.spdx.model import (
+    Actor,
+    ActorType,
+    Annotation,
+    AnnotationType,
+    Checksum,
+    ChecksumAlgorithm,
+    CreationInfo,
+    Document,
+    ExternalPackageRef,
+    ExternalPackageRefCategory,
+    Package,
+    PackagePurpose,
```

Comment:
Couldn't find any reference to `PackagePurpose`, so I assume you can get rid of it

Reply:
Good catch, thank you! I just fixed this